### PR TITLE
Gutenboarding: Add feature picker UI.

### PR DESCRIPTION
**Pull request by yansern**
## Changes proposed in this Pull Request

UI for feature picker. Just the UI, no mechanics. Currently hiding behind the `gutenboarding/feature-picker` feature flag.

## Testing instructions

* Go to [/new/features?flags=gutenboarding/feature-picker](https://hash-aaa67b719388270ede5018d89ba780e9435ae5f3.calypso.live/new/features?flags=gutenboarding/feature-picker)
* Test it out on different viewports.

### CODE REVIEW

File: client/landing/gutenboarding/onboarding-block/features/index.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+					{ hasSelectedFeatures ? (
+						<NextButton onClick={ goNext } />
+					) : (
+						<SkipButton onClick={ goNext } />
+					) }
+				</ActionButtons>
+			</div>
+			<div className="features__body">
+				<div className="features__items">
+					{ featureList.map( ( feature ) => (
```
Comment (razvanpapadopol) [inline client/landing/gutenboarding/onboarding-block/features/index.tsx:67]: We could render here `<FeatureList selected={selectedFeatures} onFeatureToggle={toggleFeature} />` so we can extract the mapping and HTML as a component to [feature-list.tsx](https://github.com/Automattic/wp-calypso/pull/44382/files#diff-5eb26386e70485c0b2a8e47dc4be6756) file.

File: client/landing/gutenboarding/onboarding-block/features/feature-list.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+					clipRule="evenodd"
+					d="M2 12C2 6.47715 6.47715 2 12 2C14.6522 2 17.1957 3.05357 19.0711 4.92893C20.9464 6.8043 22 9.34784 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12ZM16 12L10 7.5V16.5L16 12Z"
+				/>
+			</SVG>
+		),
+	},
+	{
+		id: 'support',
+		name: 'Priority Support',
+		description: 'Chat with an expert live.',
```
Comment (razvanpapadopol) [inline client/landing/gutenboarding/onboarding-block/features/feature-list.tsx:113]: Should translate `name` and `description`.

Comment (yansern) [reply client/landing/gutenboarding/onboarding-block/features/feature-list.tsx:113]: Alright! Will be moved to data stores and will be translated there.

File: client/landing/gutenboarding/onboarding-block/features/index.tsx (hunk 1 above)
Comment (yansern) [reply client/landing/gutenboarding/onboarding-block/features/index.tsx:67]: Sounds like a nice way to reuse the component.

**Comment by yansern** [comment]
> **Suggested changes:** 
> ~Domains~ Custom domains
> ~SEO~ SEO tools
> ~Remove Ads~ Ad-free
> ~Priority Support~ Priority support

Thanks! To be updated when feature list is moved to data stores.

**Comment by yansern** [comment]
> There is also a visual glitch when focusing items using TAB: https://cloudup.com/cfC7h9RIP1t

To be fixed in next PR.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4128401

Thank you @yansern for including a screenshot in the description! This is really helpful for our translators.

**Pull request merged**